src/models/question.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const commentSchema = new Schema(
+  {
+    content: {
+      type: String,
+      required: true,
+    },
+    question: {
+      type: Schema.Types.ObjectId,
+      ref: "Student",
```

Comment:
Is this field is the student field or a question one? since it's called question but referencing a student